packages/extension-dapp/src/bundle.ts

```diff
@@ -177,6 +177,43 @@ export async function web3AccountsSubscribe (cb: (accounts: InjectedAccountWithM
   };
 }
 
+export async function web3ExtensionAccountsSubscribe (extensions: string[], cb: (accounts: InjectedAccountWithMeta[]) => void | Promise<void>, { ss58Format }: Web3AccountsOptions = {}): Promise<Unsubcall> {
```

Comment:
A question on this - why don't we just add an optional `extensions?: string[]` param to `web3AccountsSubscribe`? The two are equivalent, just with an additional filter. So would suggest something like -

```js
web3AccountsSubscribe (cb: (accounts: InjectedAccountWithMeta[]) => void | Promise<void>, { extensions, ss58Format }: Web3AccountsOptions = {}) {
...
const unsubs = (await web3EnablePromise)
  .filter(({ name }) =>
    !extensions || extensions.includes(name)
  )
  .map(
    ({ accounts: { subscribe }, name: source }): Unsubcall =>
```

Does that work for you?

Reply:
This works very well. I thought we might want to keep extensions out of account options, but this is more DRY and makes sense to have just an extra option there.

Reply:
Perfect, all happy with that approach for the same DRY reason. We can add the same to web3Accounts then as well (makes it consistent, it anyway uses the same options interface).